JACP.JavaFX/src/main/java/org/jacpfx/rcp/handler/ComponentHandlerImpl.java

```diff
@@ -157,7 +157,7 @@ private void putMessageToQueue(final Message<Event, Object> action,
 
 	private void log(final String message, final String name) {
 		if (this.logger.isLoggable(Level.FINE)) {
-			this.logger.fine(">> " + message+ ' ' +name);
+			this.logger.finest(">> " + message+ ' ' +name);
```

Comment:
Isn't this a limitation of our approach. It is another case of a wrapped log.

Reply:
I agree. I will add it into paper.

Reply:
@saledouble Can you reference the paper commit?